src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -8,6 +8,10 @@
     [org.broadinstitute.firecloud-ui.common.table :as table]
     [org.broadinstitute.firecloud-ui.common.table-utils :as table-utils]
     [org.broadinstitute.firecloud-ui.common.icons :as icons]
+    [org.broadinstitute.firecloud-ui.common.modal :as modal]
+    [org.broadinstitute.firecloud-ui.page.workspace.data.import-data :as import-data]
+    [org.broadinstitute.firecloud-ui.utils :as utils :refer [access-token]]
```

Comment:
Don't need this line (see comment below). Also, alphabetize imports so we don't have to think about the order.